README.md

```diff
@@ -126,15 +126,15 @@ We use a publicly attested event so that you can check the event details directl
 
 **By default there will be no output. If the verification has succeeded an exit code of 0 will be returned.**
 
-If the verification command is run with `--log-level=INFO` the output will be:
+If the verification command is run with `--loglevel=INFO` the output will be:
 
     verifying for tenant: tenant/6ea5cd00-c711-3649-6914-7b125928bbb4
     verifying: 663 334 018fa97ef269039b00 2024-05-24T08:27:00.2+01:00 publicassets/87dd2e5a-42b4-49a5-8693-97f40a5af7f8/events/a022f458-8e55-4d63-a200-4172a42fc2aa
     leaf hash: bfc511ab1b880b24bb2358e07472e3383cdeddfbc4de9d66d652197dfb2b6633
     OK|663 334|[aea799fb2a8..., proof path nodes, ...f0a52d2256c235]
 
 
-The elided proof path nodes will be:
+The elided proof path at time of writing was:
 
     [9f0183c7f79fd81966e104520af0f90c8447f1a73d4e38e7f2f23a0602ceb617,
```

Comment:
think the proof path is also wrong as it's missing the first hash no?

Reply:
Pushed change

Reply:
it's actually 

```[aea799fb2a8c4bbb6eda1dd2c1e69f8807b9b06deeaf51b9e0287492cefd8e4c, 9f0183c7f79fd81966e104520af0f90c8447f1a73d4e38e7f2f23a0602ceb617, da21cb383d63896a9811f06ebd2094921581d8eb72f7fbef566b730958dc35f1, 51ea08fd02da3633b72ef0b09d8ba4209db1092d22367ef565f35e0afd4b0fc3, 185a9d55cf507ef85bd264f4db7228e225032c48da689aa8597e11059f45ab30, bab40107f7d7bebfe30c9cea4772f9eb3115cae1f801adab318f90fcdc204bdc, 94ca607094ead6fcd23f52851c8cdd8c6f0e2abde20dca19ba5abc8aff70d0d1, ba6d0fd8922342aafbba6073c5510103b077a7de9cb2d72fb652510110250f9e, 7fafc7edc434225afffc19b0582efa2a71b06a2d035358356df0a52d2256c235, b737375d837e67ee7bce182377304e889187ef0f335952174cb5bf707a0b4788]```